scripts/Sign-Package.ps1

```diff
@@ -0,0 +1,24 @@
+$currentDirectory = split-path $MyInvocation.MyCommand.Definition
+
+# See if we have the ClientSecret available
+if([string]::IsNullOrEmpty($env:SignClientSecret)){
+	Write-Host "Client Secret not found, not signing packages"
+	return;
+}
+
+dotnet tool install --tool-path . SignClient
+
+# Setup Variables we need to pass into the sign client tool
+$appSettings = "$currentDirectory\SignClient.json"
+
+$nupgks = ls $Env:ArtifactDirectory\*.nupkg | Select -ExpandProperty FullName
+
+foreach ($nupkg in $nupgks){
+	Write-Host "Submitting $nupkg for signing"
+
+	.\SignClient 'sign' -c $appSettings -i $nupkg -r $env:SignClientUser -s $env:SignClientSecret -n 'Portable.BouncyCastle' -d 'Portable.BouncyCastle' -u 'https://github.com/onovotny/bc-sharp' 
```

Comment:
I think you should prepend this line with a `&`

Reply:
Why?

Reply:
https://ss64.com/ps/call.html
It's the call operator for powershell - sure most of the time it will work without it - but it's just a good habit.